src/transforms/transform-pull-request.ts

```diff
@@ -19,29 +19,44 @@ const mapStatus = (status: string, merged_at?: string) => {
 };
 
 // TODO: define arguments and return
-const mapReviews = (reviews: Octokit.PullsListReviewsResponse = []) => {
+const mapReviews = async (reviews: Octokit.PullsListReviewsResponse = [], gitHubInstallationClient: GitHubInstallationClient): Promise<JiraReview[]> => {
 	const sortedReviews = orderBy(reviews, "submitted_at", "desc");
 	const usernames: Record<string, JiraReview> = {};
-	// The reduce function goes through all the reviews and creates an array of unique users (so users' avatars won't be duplicated on the dev panel in Jira) and it considers 'APPROVED' as the main approval status for that user.
-	return sortedReviews.reduce((acc: JiraReview[], review) => {
+	// The reduce function goes through all the reviews and creates an array of unique users
+	// (so users' avatars won't be duplicated on the dev panel in Jira)
+	// and it considers 'APPROVED' as the main approval status for that user.
+	const reviewsReduced: JiraReview[] = sortedReviews.reduce((acc: JiraReview[], review) => {
 		// Adds user to the usernames object if user is not yet added, then it adds that unique user to the accumulator.
 		const author = review?.user;
+
 		if (!usernames[author?.login]) {
 			usernames[author?.login] = {
 				...getJiraAuthor(author),
+				login: author.login,
 				approvalStatus: review?.state === "APPROVED" ? "APPROVED" : "UNAPPROVED"
 			};
+
 			acc.push(usernames[author?.login]);
-			// If user is already added (not unique) but the previous approval status is different than APPROVED and current approval status is APPROVED, updates approval status.
+			// If user is already added (not unique) but the previous approval status is different
+			// from APPROVED and current approval status is APPROVED, updates approval status.
 		} else if (
 			usernames[author?.login].approvalStatus !== "APPROVED" &&
 			review.state === "APPROVED"
 		) {
 			usernames[author?.login].approvalStatus = "APPROVED";
 		}
+
 		// Returns the reviews' array with unique users
 		return acc;
 	}, []);
+
+	return Promise.all(reviewsReduced.map(async reviewer => {
+		const gitHubUser = await getGithubUser(gitHubInstallationClient, reviewer.login);
+		return {
+			...reviewer,
+			email: gitHubUser && gitHubUser.email ?  gitHubUser.email : `${reviewer.login}@noreply.user.github.com`
```

Comment:
could be simplified to `gitHubUser?.email || `${reviewer.login}@noreply.user.github.com``